src/app/legacy/containers/Navigation/__snapshots__/index.test.jsx.snap

```diff
@@ -317,6 +317,7 @@ exports[`Navigation Container should correctly render amp navigation 1`] = `
   right: 0;
   bottom: 0;
   border-bottom: 0.25rem solid #B80000;
+  border-bottom: 0.3125rem solid #B80000;
```

Comment:
Not sure what's causing this yet

Reply:
Likely related to https://github.com/bbc/simorgh/pull/10712/#discussion_r1146062613